runtime/vm/jvminit.c

```diff
@@ -4218,6 +4218,18 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		} else if (flightRecorder < noFlightRecorder) {
 			vm->extendedRuntimeFlags2 &= ~(UDATA)J9_EXTENDED_RUNTIME2_JFR_ENABLED;
 		}
+
+		IDATA jfrOptionIndex = FIND_AND_CONSUME_VMARG(OPTIONAL_LIST_MATCH_USING_EQUALS, VMOPT_XXSTARTFLIGHTRECORDING, NULL);
+		if (jfrOptionIndex >= 0) {
+			char* jfrOptionBuffer = NULL;
+			GET_OPTION_VALUE(jfrOptionIndex, '=', &jfrOptionBuffer);
+			if(jfrOptionBuffer == NULL) {
+				vm->jfrState.jfrCMDLineOption = (char*)"dumponexit=false";
```

Comment:
does this mean JFR is always enabled? see `initJFR`

Reply:
In JDK17, Just specifying `-XX:StartFlightRecording` will enable the JFR.